content/rc/administration/security/data-access-control.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Data Access Control
+description:
+weight: 40
+alwaysopen: false
+categories: ["RC"]
+---
+Role-Based Access Control (RBAC) lets you scale your Redis deployments while simplifying the complexity of managing data access for a cluster with many databases, users, and access control lists.
+With RBAC, you can create a role for data access and apply it to many users to define their access to multiple databases in the cluster.
+
+In **Data Access Control** > **Roles**, you can configure data access roles with Redis ACLs that define the commands and keys that users can access in database connections
+
+To control user access to Redis database commands and keys,
+you must define Redis ACLs that specify the commands that users can run and keys that the commands can apply to.
+Then, in the data access role, select the databases that the users can access and Redis ACL that controls user access to those databases.
+
+A-A cannot assign ACLs on creation
+
+{{% note %}}
+
+- Redis ACLs can only be configured in the cluster web UI or API.
+    In Redis:
+    - These ACL subcommands are blocked: LOAD, SAVE, SETUSER, DELUSER, GENPASS, LOG
+    - These ACL subcommands are allowed: LIST, USER, GETUSER, CAT, WHOAMI, HELP
+- The MULTI, EXEC, DISCARD commands are always allowed, but ACLs are enforced on MULTI subcommands.
+- When you run multi-key commands on multi-slot keys, the return value is `failure` but the command runs on the keys that are allowed.
+
+{{% /note %}}
+
+## Redis ACL command syntax
+
+Redis ACLs are defined by a [Redis syntax](https://redis.io/topics/acl#acl-rules) where you specify the commands or command categories that are allowed for specific keys.
+A command category is a predefined, named set of commands that perform a function, for example `read` commands or `dangerous` commands.
+You can also define Redis ACLs with module commands for any modules that are loaded on the cluster.
+If you run a command on multiple databases including databases where the command is not allowed by ACLs or the command does not exist,
+the command succeeds where possible.
+
+A Redis ACL syntax lets you:
+
+- Include commands and categories with the `+` or exclude commands and categories with the `-` prefix
+- Define categories with the `@` prefix
+- Define keys or key patterns with the `~` prefix
+
+The predefined Redis ACLs are:
```

Comment:
s/ACLs/ACL rules/

Reply:
The terminology of "rules" is not used in the UI so I would prefer to avoid it.